assets/css/Useful-links.css

```diff
@@ -139,8 +139,8 @@ h1 {
 }
 
 .list {
```

Comment:
Really good changes! This makes the links respond much better to different screen size.